changelog.yaml

```
@@ -59,6 +59,15 @@
         The `master` branch of repo `stewardci-example-pipelines`
         has been renamed to `main` which made the examples fail.
       pullRequestNumber: 257
+    - type: internal
+      impact: patch
```

This is not an `internal` change, but an `enhancement`. I would also classify it as minor change.

```suggestion
    - type: enhancement
      impact: minor
```